### PR TITLE
Fix not hiding tokens for clients via realtime

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Fix secret being set in realtime payload

## Test Plan

Existing tests

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/api/account.php [hunk 1]
```diff
@@ -1845,6 +1845,9 @@
             'team' => '',
         ];
 
+        // Hide secret for clients
+        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $tokenSecret : '');
```
Comment (stnguyen90) [inline app/controllers/api/account.php:1849]: This will remove the secret from other destinations like functions and webhooks, too won't it?

Comment (abnegate) [reply app/controllers/api/account.php:1849]: Yep, will fix

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Did the tests break?

**Review by stnguyen90 — APPROVED** [review]
looks good. there's just a merge conflict.

**Pull request merged**